convert/src/label_studio_to_badgerdoc/badgerdoc_format/pdf_converter.py

```diff
@@ -0,0 +1,52 @@
+from typing import List
+
+from pdfminer.high_level import extract_pages
+from pdfminer.layout import LTChar, LTTextBoxHorizontal, LTTextLineHorizontal
+
+from src.label_studio_to_badgerdoc.models import (
+    BadgerdocToken,
+    Offset,
+    Page,
+    PageSize,
+)
+from src.label_studio_to_badgerdoc.models.bd_tokens_model import Pages
+
+
+class PlainPDFToBadgerdocTokensConverter:
+    def __init__(self):
+        self.offset = 0
```

Comment:
it could be offset = 0 without __init__method.
Initializer does not take any arguments

Reply:
This one is not fixed